src/mips/openbios/main/main.c

```diff
@@ -75,8 +80,9 @@ int main() {
     POST = 0x0f;
     muteSpu();
 
-    if (checkExp1PreHookLicense()) runExp1PreHook();
+    runExp1PreHook();
```

Comment:
Maybe add a comment here that this is a change from the retail bios in that the check is folded in the function, whereas in retail it's here. Don't forget the code is also meant as a reference to how the retail code is laid out, so we need to comment deviations like that.